pom.xml

```diff
@@ -6,13 +6,13 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>146</version>
+        <version>147</version>
```

Comment:
Please mention this update in the a commit message or do it in a separate commit

Reply:
The commit body already mentions: 
```
Also, upgrade airbase to 147 that contains sortpom-maven-plugin.
```